.pylintrc

```diff
@@ -63,6 +63,8 @@ indent-string='    '
 
 allowed-comment-prefixes=int,str,float,bool,v
 
+disallowed-functions=now=>datetime.datetime.utcnow,assertEquals=>self.assertEqual,StringIO=>python_utils.string_io,urlsplit=>python_utils.url_split,urlparse=>python_utils.url_parse,urlunsplit=>python_utils.url_unsplit,parse_qs=>python_utils.parse_query_string,unquote=>python_utils.urllib_unquote,urljoin=>python_utils.url_join,next=>python_utils.NEXT,range=>python_utils.RANGE,round=>python_utils.ROUND,str=>python_utils.convert_to_bytes or python_utils.UNICODE,zip=>python_utils.ZIP,basestring=>python_utils.BASESTRING,iteritems=>items,itervalues=>values,iterkeys=>keys,put_async=>put,put_multi_async=>put_multi
```

Comment:
maybe, `disallowed-functions-with-alternatives` or `replacable-dissalowed-functions`?

Also, for the function which are disallowed and have no alternatives can we use `[bad-builtin](https://github.com/PyCQA/pylint/blob/548b38e0508cf58ee9c2fb2e92c0fbb78a1774ef/tests/extensions/test_bad_builtin.py)` checker?

Reply:
Will rename the config. For those without alternatives, I think that's okay, but could we do that on a separate issue/PR?

Reply:
>  I think that's okay, but could we do that on a separate issue/PR

Yep, I agree, can you create an issue to track this task, and let's do it after this PR? :)